health/REFERENCE.md

```diff
@@ -129,6 +132,32 @@ You're interested in what comes after the comma: `disk.io`. That's the name of t
 
 If you create a template using the `disk.io` context, it will apply an alarm to every disk available on your system.
 
+#### Alarm line `class`
+
+Specify the classification of the alarm or template. 
+
+Class can be used to indicate the broader area of the system that the alarm applies to. For example, under the general `Database` class, you can group together alarms that operate on various database systems, like `MySQL`, `CockroachDB`, `CouchDB` etc.
+
+```yaml
+class: Database
+```
+
+#### Alarm line `component`
+
+Component can be used to narrow down what the previous `class` value specifies for each alarm or template. Continuing from the previous example, `component` might include `MySQL`, `CockroachDB`, `MongoDB`, all under the same `Database` classification.
+
+```yaml
+component: MySQL
+```
+
+#### Alarm line `type`
+
+This indicates the type of error (or general problem area) that the alarm or template applies to. For example, `Latency` can be used for alarms that trigger on latency issues, be it in network interfaces, web servers, or database systems.
+
+```yaml
+type: Latency
+```
```

Comment:
Right now our current code gives a big freedom for users, I do not see this as a problem, on the other hand this freedom also brings the necessity to start thinking on binary trees to get data related to these possible values, I did not see them in the previous PR.

Reply:
According your previews [answer](https://github.com/netdata/netdata/pull/10984#discussion_r614888815) this is liked to be used on cloud only, so I will mark this as resolved, but case we start using this on agent, it will be necessary to add more control on agent side.